src/main/java/vazkii/botania/api/corporea/CorporeaHelper.java

```diff
@@ -25,6 +29,11 @@
 import net.minecraft.world.World;
 import net.minecraftforge.common.MinecraftForge;
 import net.minecraftforge.common.util.ForgeDirection;
+import vazkii.botania.api.BotaniaAPI;
+import vazkii.botania.common.Botania;
+import vazkii.botania.common.integration.corporea.WrappedDeepStorage;
+import vazkii.botania.common.integration.corporea.WrappedIInventory;
+import vazkii.botania.common.integration.corporea.WrappedStorageDrawers;
```

Comment:
Need to get @Vazkii to confirm this but I think there will be a problem here with calling into the main Botania code from the API, afaik this will cause problems for other people who are using the API